ATTRIBUTION.md

```diff
@@ -333,4 +333,63 @@ The dataset is used to generate wind profiles from wind speed profiles.
 ##### Note
 Recommended citation: J. King, A. Clifton and B.-M. Hodge. 2014. Validation of Power Output for the WIND Toolkit, Golden, CO: National Renewable Energy Laboratory. NREL/TP-6A20-74110. https://www.nrel.gov/docs/fy14osti/61714.pdf
 
----
\ No newline at end of file
+
+---
+### Grid
+#### Homeland Infrastructure Foundation-Level Data (HIFLD)
+##### Source
+* Name: Electric Power Transmission Lines
+* Author: HIFLD Open
+* Description: This dataset represents electric power transmission lines. Lines are operated at relatively high voltages varying from 69 kV up to 765 kV, and are capable of transmitting large quantities of electricity over long distances.
+* Source: https://hifld-geoplatform.opendata.arcgis.com/
+* Exact source location: https://hifld-geoplatform.opendata.arcgis.com/datasets/electric-power-transmission-lines/explore?showTable=true
+
+##### Destination
+* Modifications to source file(s): None
+* Location:
+  * https://besciences.blob.core.windows.net/datasets/hifld/Electric_Power_Transmission_Lines_Jul2020.csv
+  * https://besciences.blob.core.windows.net/datasets/hifld/Electric_Power_Transmission_Lines_Jul2020.geojson.zip
+
+##### General Purpose
+The data are used to populate the transmission topology of U.S. power grid that can be used to run scenarios.
+
+##### Note
+Public use (see https://hifld-geoplatform.opendata.arcgis.com/datasets/geoplatform::electric-power-transmission-lines/about). Users are advised to read the data set's metadata thoroughly to understand appropriate use and data limitations.
+
+
+##### Source
+* Name: Electric Substations
+* Author: HIFLD Open
+* Description: This dataset represents electric power substations primarily associated with electric power transmission. In this layer, substations are considered facilities and equipment that switch, transform, or regulate electric power at voltages equal to, or greater than, 69 kilovolts.
```

Comment:
Should we use kV instead of kilovolts to be consistent here.

Reply:
Done